tests/api/test_checkout.py

```diff
@@ -978,28 +978,6 @@ def test_checkout_shipping_address_update_without_phone_country_prefix(
     assert not data["errors"]
 
 
-def test_checkout_shipping_address_update_invalid_country_code(
-    user_api_client, checkout_with_item, graphql_address_data
-):
-    checkout = checkout_with_item
-    assert checkout.shipping_address is None
-    checkout_id = graphene.Node.to_global_id("Checkout", checkout.pk)
-
-    shipping_address = graphql_address_data
-    shipping_address["country"] = "CODE"
-    variables = {"checkoutId": checkout_id, "shippingAddress": shipping_address}
-
-    response = user_api_client.post_graphql(
-        MUTATION_CHECKOUT_SHIPPING_ADDRESS_UPDATE, variables
-    )
-    content = get_graphql_content(response)
-    data = content["data"]["checkoutShippingAddressUpdate"]
-    assert data["errors"][0]["message"] == "Invalid country code."
-    assert data["errors"][0]["field"] == "country"
-
-    assert data["checkoutErrors"][0]["code"] == CheckoutErrorCode.INVALID.name
-
-
```

Comment:
I'm guessing we don't need the attached code below anymore? https://github.com/mirumee/saleor/blob/d369a4bce011d86c1ac7bae37ca5314f96a84259/saleor/graphql/account/i18n.py#L32-L41